sktime/regression/tests/test_all_regressors.py

```diff
@@ -71,12 +71,11 @@ def test_regressor_output(self, estimator_instance, scenario):
         # we use check_is_scitype to get the number instances in X_new
         #   this is more robust against different scitypes in X_new
         _, _, X_new_metadata = check_is_scitype(X_new, "Panel", return_metadata=True)
-        X_new_instances = X_new_metadata["n_instances"]
```

Comment:
why are we changing this?

Reply:
`X_meta_instances` no longer needed, I modified another line below. I shall add it back if it's required.

Reply:
I meant, we still should check for the right length here, i.e., leave it in - I do not see how the expected output format would change in the scenario.

Reply:
Yeah, I see it. Now the shape check should be modified according to dim of y i.e., one case is when dim is 1, and another when dim is 2.